scrapelib/__init__.py

```diff
@@ -543,9 +544,9 @@ def request(  # type: ignore
         # for example 'HIGH:!DH:!aNULL' to bypass "dh key too small" error
         # https://stackoverflow.com/questions/38015537/python-requests-exceptions-sslerror-dh-key-too-small
         if ciphers_list_addition:
-            requests.packages.urllib3.util.ssl_.DEFAULT_CIPHERS += ciphers_list_addition  # type: ignore
+            urllib3.util.ssl_.DEFAULT_CIPHERS += ciphers_list_addition  # type: ignore
```

Comment:
`requests.packages.urllib3` is an alias for `urllib3`, so I use the lib directly to shut pylint up. Reference: https://github.com/pylint-dev/pylint/issues/4584